pkg.json: `main` points to non existent file

https://github.com/prisma/prisma-fmt-wasm/blob/03ea440e552502258a2c047ef8e0ae8432e61309/package.json#L5

```
"main": "src/index.js",
```

Is this a problem?

I think it is:

```
const prismaFmt = require('@prisma/prisma-fmt-wasm')
console.log(prismaFmt.version())
```

leads to

```
C:\Users\Jan\Documents\throwaway\prisma-fmt-wasm>node script.js
internal/modules/cjs/loader.js:328
      throw err;
      ^

Error: Cannot find module 'C:\Users\Jan\Documents\throwaway\prisma-fmt-wasm\node_modules\@prisma\prisma-fmt-wasm\src\index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:320:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:533:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:875:27)
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (C:\Users\Jan\Documents\throwaway\prisma-fmt-wasm\script.js:1:19)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32) {
  code: 'MODULE_NOT_FOUND',
  path: 'C:\\Users\\Jan\\Documents\\throwaway\\prisma-fmt-wasm\\node_modules\\@prisma\\prisma-fmt-wasm\\package.json',
  requestPath: '@prisma/prisma-fmt-wasm'
}
```

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "@prisma/prisma-fmt-wasm",
   "version": "3.5.0-25.0e4d0a36c14e3fcf6988f879c82f96a8f0aae54c",
   "description": "",
-  "main": "src/index.js",
+  "main": "src/prisma_fmt_build.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },

